### ISSUE TITLE
Add Windows 10 BSOD

### ISSUE THREAD

**Issue by sanyer** [dessibelle/Blue-Screen-Saver#2]
Would be nice to have Windows 10 BSOD: https://www.windowscentral.com/sites/wpcentral.com/files/styles/xlarge/public/field/image/2017/02/windows-10-blue-screen-death.jpg

**Comment by dessibelle** [comment]
Good idea @sanyer! I've been thinking about splitting the current implementation in `BSODSaverView` into two separate classes, one for Win XP and one for Win 95/98, so once that's done it should be fairly easy to add one for Win 10 as well.

I'm not that familiar with Windows - how about Win Vista, 7 and 8, do they look the same as the screenshot you posted? Also, do you by any change know what font that is?

**Comment by sanyer** [comment]
As far as I remember, Windows Vista and 7 used same BSOD style as XP and Windows 8 had a bit simpler version of one used in Windows 10 without QR code.

There are also a lot of errors which would be nice to randomize on every run: https://www.lifewire.com/blue-screen-error-codes-4065576

**Comment by dessibelle** [comment]
Thanks! Yes, randomized error codes and addresses are fun. There are already a couple of them in the current implementation, but that list looks like a good resource for adding a couple more.

**Comment by akphi** [comment]
@dessibelle sorry, this is a little bit off topic, but could you make it so that this one can be installed without bypassing gatekeeper? It keeps mentioning "unidentified developer" on my Mac

**Comment by dessibelle** [comment]
@blacksteed232 Yes, that sounds reasonable. I think the build that's linked to now is a quite old build without any developer certificate. I'll try to see to that the next time I make a build.

In the mean time I think the easiest solution is that you clone the repo and build it yourself. Then you should be able to make a build with a valid certificate, which you then could install without bypassing Gatekeeper.

**Comment by ghost** [comment]
Here a PNG for win10 BSOD "critical process died" : [bsod.png](https://github.com/mancoast/BSOD-ScreenSaver/blob/master/img/bsod.png). In the [img](https://github.com/mancoast/BSOD-ScreenSaver/blob/master/img) folder there are all sorts of goodies for emulating the entire boot sequence leading up to the dreaded windows 10 hardware "crash"

**Comment by dessibelle** [comment]
Thanks @mancoast! I have not had time to look into this yet. If anyone else wants to contribute with a PR that'd be much obliged :)

**Comment by Xeiro** [comment]
Please add Win10 if you will have time for it :)

**Comment by ghost** [comment]
Will do, I am curious which is build broke the functionality. What’s is 1803?

**Comment by dessibelle** [comment]
>Will do, I am curious which is build broke the functionality. What’s is 1803?

Not sure I understand @coast77777777777. If you are referring to "which macOS build" then the only one I've persoanlly tried is `19A583`.

**EDIT**: Probably confusing this with [another issue](https://github.com/dessibelle/Blue-Screen-Saver/issues/6), but that said I still don't understand your last message.
